### ISSUE TITLE
Module 'rouge' doesn't exist on the Hugging Face Hub either.

### ISSUE THREAD

**Issue by Ysy0219** [abman23/seq2seq-SC#2]
Hello teacher, I have the following problem now.
WARNING:tensorflow:Gradients do not exist for variables when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss` argument?
May I ask how I can solve the above problem！Thank you very much！
